build/litsearch/combineDataSets.py

```diff
@@ -0,0 +1,96 @@
+import pandas as pd
+import os
+import urllib.request
+import requests
+import datetime as dt
+
+def update_AI(fname):
+    # Download most up-to-date metadata from Allen AI COVID-19 dataset
+    url = 'https://ai2-semanticscholar-cord-19.s3-us-west-2.amazonaws.com/latest/metadata.csv'
+    urllib.request.urlretrieve(url, filename=fname)
+
+def clean_df(df):
+    df = df.dropna(subset=["doi"])
```

Comment:
Depending on how the resulting `covid19-lit-all.tsv` file ends up being used, we will want to keep in mind that sources without DOIs have been removed and may want to consider adding them back in.

Reply:
@rdvelazquez that is a very good point! For @danich1 analysis I think we need to drop them, which means they wouldn't be among the sources suggested for each topic (but it would be hard to suggest them without being able to pull text). For anything involving internal curation, we should definitely stick with the original output from your script.